stream/src/main/scala/libretto/stream/CoreStreams.scala

```diff
@@ -18,7 +18,7 @@ class CoreStreams[DSL <: CoreDSL, Lib <: CoreLib[DSL]](
 ) {
   import dsl._
   import dsl.$._
-  import lib._
+  import lib.{_, given}
```

Comment:
I think on this level it makes sense. But not sure.

Like said, this needs some rethinking in general, I guess.